perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/scheduling/impl/SchedulingPoolImpl.java

```diff
@@ -146,7 +147,24 @@ public void setPerun(Perun perun) {
 		this.perun = perun;
 	}
 
-
+	// --- session init ----------------------------------
+	
+	public void afterPropertiesSet() {
```

Comment:
This method is not used anywhere.

Reply:
This method is called by Spring after bean creation and initialization. It is implementation of InitializingBean interface.